### PR TITLE
Add IE11 support

### LINKED ISSUE #37: Error in IE11: URLSearchParams is undefined
One of our Altis sites requires IE11 support, and this module utilizes the [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams) interface which is not available in that environment.

It looks like there's at least [one extant polyfill](https://www.npmjs.com/package/url-search-params-polyfill), though I haven't vetted it yet. Alternatively we could accept that analytics will not work properly in IE11 because it won't have _that_ much marketshare, but it shouldn't be too onerous to fix properly.

### PR DESCRIPTION
**Pull request by roborourke**
Fixes #37

This also simplifies and removes duplication of some of the complex es6 syntax around building the attributes objects to send with events.

### CODE REVIEW

**Comment by roborourke** [comment]
@kadamwhite bumping this, any chance of a review?

**Comment by roborourke** [comment]
Will wait for #47 to go in first as there's some overlap regarding the `getAttributes()` and `getMetrics()` functions.

**Comment by roborourke** [comment]
Updated to handle all query vars the same way, they were needlessly duplicated before.

File: src/analytics.js [hunk 1]
```diff
@@ -62,16 +69,13 @@ window.addEventListener( 'scroll', () => {
...
-const params = new URLSearchParams( window.location.search );
-const utm = {
-	utm_source: params.get( 'utm_source' ) || '',
-	utm_medium: params.get( 'utm_medium' ) || '',
-	utm_campaign: params.get( 'utm_campaign' ) || '',
-	utm_term: params.get( 'utm_term' ) || '',
-	utm_content: params.get( 'utm_content' ) || '',
-};
+const params = parseQueryString( window.location.search );
+const qv_params = {};
```
Comment (kadamwhite) [inline src/analytics.js:75]: nitpick: `qvParams`?

File: src/utils/index.js [hunk 2]
```diff
@@ -32,3 +32,59 @@ export const throttle = ( delay, fn ) => {
...
+	}
+	if ( ! Array.isArray( value ) ) {
+		value = [ value ];
+	}
+	return value.map( sanitizeCallback );
+};
+
+const sanitizeAttribute = value => value.toString();
+
+const sanitizeMetric = value => parseFloat( Number( value ) );
```
Comment (kadamwhite) [inline src/utils/index.js:62]: It's pretty obvious what these do but docbloc presence is inconsistent in this file

File: src/analytics.js (hunk 1 above)
Comment (roborourke) [reply src/analytics.js:75]: Yep, don't know I had a blank on that one!

**Comment by roborourke** [comment]
Thanks @kadamwhite, much appreciated

**Pull request merged**